### DISCUSSION TITLE
Having a Tailwindcss plugin converter like windy

### DISCUSSION THREAD

**Discussion by riderx** [unocss/unocss#2462] [Ideas]
I know the concept of preset in Uno is a bit different than Tailwind.

Right now the biggest issue for me to switch is that my plugin are not available.
KonstaUI, daisy, flowbite.

I'm sure i'm not the only one.

I'm willing to do the job and make it happen, i just need a bit on guidance of the differences right now i cannot find much documentation of the difference of both.

I truely believe Uno will become bigger than tailwind because the DX is way above, we just need to make switch a no brainer

**Comment by InsaneNaman** [comment]
I am exploring to switch to Uno.css but not finding the right documentation to consume tailwind plugin. May be I am missing something?

**Comment by chu121su12** [comment]
https://unocss.dev/guide/why/

> The downside of UnoCSS over Tailwind CSS is that it does not support Tailwind's plugin system or configurations, meaning it might make it harder to migrate from a heavily customized Tailwind CSS project. This is the intended decision to make UnoCSS high-performant and extensible, and we believe the trade-off is worth it.

**Reply by chu121su12** [reply]
We do really appreciate contribution to make migration or migration steps easier
